packages/plugins/rubygem-smart_proxy_pulp/rubygem-smart_proxy_pulp.spec

```diff
@@ -6,23 +6,18 @@
 
 Summary: Basic Pulp support for Foreman Smart-Proxy
 Name: rubygem-%{gem_name}
-Version: 1.3.0
+Version: 1.4.0
 Release: 1%{?dist}
 Group: Applications/System
 License: GPLv3
 URL: https://github.com/theforeman/smart-proxy-pulp
```

Comment:
Bonus points if you use the direct URL (https://github.com/theforeman/smart_proxy_pulp)